package.json

```diff
@@ -12,20 +12,22 @@
     "Maël Thomas-Quillévéré <mael.thomas@data.gouv.fr>"
   ],
   "description": "Un simulateur de coût d'embauche",
-  "engines": { "node": ">=0.12" },
+  "engines": {
+    "node": ">=0.12"
```

Comment:
And me with Node 4.2.1, ok  ;)

Would still be safer to set this to `>=0.12 <=5` to prevent false forward-compat, WDYT?